test/issue103-single-file-package.sh

```diff
@@ -1,6 +1,18 @@
 #!/bin/sh
 set -e
 cd ${CURR_DIR}
+rm -f single-file-test
 ./issue103-single-file-package.d foo -- bar
+if [ -f single-file-test ]; then
+	echo "Shebang invocation produced binary in current directory"
+	exit 1
+fi
```

Comment:
IIUC, the binary will placed in the parent directory. Can you verify that with a test?